src/engine/bank06.asm

```diff
@@ -3392,23 +3392,23 @@ HandleCardPopCommunications:
 	or a
 	jr z, .success
 	; not $00, means the name was found in the list
-	ldtx hl, Text025c ; CannotCardPopWithFriendPreviouslyPoppedWithText
+	ldtx hl, CannotCardPopWithFriendPreviouslyPoppedWithText
 	ld a, [wCardPopType]
 	cp $01
 	jr z, .set_carry
-	ldtx hl, Text0261
+	ldtx hl, CannotRareCardPopWithFriendPreviouslyPoppedWithText
 	jr .set_carry
 
 .fail
 	ld a, [wIRCommunicationErrorCode]
 	cp $01
 	jr nz, .asm_19e7f
-	ldtx hl, Text0262 ; ThePopWasntSuccessfulText
+	ldtx hl, CardPopModeMismatchedText ; Card Pop from one side and Rare Card Pop from the other
 	scf
 	ret
 
 .asm_19e7f
-	ldtx hl, Text025b
+	ldtx hl, PopUnsuccessfulText
```

Comment:
We're pretty consistent about referring to it as "CardPop". Perhaps this should be `ThePopWasntSuccessfulText`?